x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -44,6 +45,11 @@ export class Simulator {
    * This is used by `debugActions`.
    */
   private readonly spyMiddleware: SpyMiddleware;
+  /**
+   * Simulator which allows you to explicitly simulate resize events and trigger animation frames
+   */
+  public readonly sideEffectSimulator: SideEffectSimulator;
```

Comment:
```suggestion
  private readonly sideEffectSimulator: SideEffectSimulator;
```
This could be private

Reply:
done